lib/sqlHandler.js

```diff
@@ -455,7 +463,8 @@ SqlStore.prototype._clearAndSetRelationTables = function (theResource, partialRe
     if (!partialResource.hasOwnProperty(relationName)) return
     var relationModel = self.relations[relationName]
 
-    var keyName = self.resourceConfig.resource + '-' + relationName
+    var delimiter = self.sequelize.options.define.underscored ? UNDERSCORED_DELIMITER : DEFAULT_RELATION_DELIMITER
```

Comment:
crummy you have to do this check twice. Maybe it could be `self.sequelize.options.define.relationDelimiter` that you set based on `...define.underscored` 1 time?

wow... haven't seen a `var self = this` is a long while. Don't miss it.

Reply:
So `self.sequelize.options.define.relationDelimiter` would be `'_'` if `...define.underscored` is `true`? I would still have to check if that property is `nil` and replace with the default if it's `nil`. It's not guaranteed to exist. I'm not really sure what this change would get us.